packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -38,29 +40,50 @@ import SignupConfirmed from '../../pages/Signup/SignupConfirmed';
 import ConfirmSignupCode from '../../pages/Signup/ConfirmSignupCode';
 import SigninReported from '../../pages/Signin/SigninReported';
 import SigninBounced from '../../pages/Signin/SigninBounced';
-import AccountRecoveryResetPassword from '../../pages/ResetPassword/AccountRecoveryResetPassword';
 import LinkValidator from '../LinkValidator';
 import { LinkType } from 'fxa-settings/src/lib/types';
 import Confirm from 'fxa-settings/src/pages/Signup/Confirm';
 import WebChannelExample from '../../pages/WebChannelExample';
 import { CreateCompleteResetPasswordLink } from '../../models/reset-password/verification/factory';
 import ThirdPartyAuthCallback from '../../pages/PostVerify/ThirdPartyAuthCallback';
+import {
+  SettingsContext,
+  initializeSettingsContext,
+} from '../../models/contexts/SettingsContext';
+import CompleteResetPasswordContainer from '../../pages/ResetPassword/CompleteResetPassword/container';
+import AccountRecoveryResetPasswordContainer from '../../pages/ResetPassword/AccountRecoveryResetPassword/container';
+import { QueryParams } from '../..';
+
+// TODO: FXA-8098
+// export const INITIAL_METRICS_QUERY = gql`
+//   query GetInitialMetricsState {
+//     account {
+//       recoveryKey
+//       metricsEnabled
+//       emails {
+//         email
+//         isPrimary
+//         verified
+//       }
+//       totp {
+//         exists
+//         verified
+//       }
+//     }
+//   }
+// `;
 
 export const App = ({
   flowQueryParams,
 }: { flowQueryParams: QueryParams } & RouteComponentProps) => {
-  const [isSignedIn, setIsSignedIn] = useState<boolean>();
-
-  const { showReactApp, isInRecoveryKeyExperiment } = flowQueryParams;
-  const { loading, error } = useInitialState();
-  const account = useAccount();
-  const [email, setEmail] = useState<string>();
-  const [emailLookupComplete, setEmailLookupComplete] =
-    useState<boolean>(false);
```

Comment:
I took the `showReactApp` check out which we were using to conditionally include routes because I don't think we need it.

I also took out some of this `emailLookup` stuff out since it looked like we were just doing the same thing as what `currentAccount` does, but either way it shouldn't affect existing functionality for now since it's for signin/signup pages.